packages/medusa/src/interfaces/csv-parser.ts

```diff
@@ -0,0 +1,34 @@
+import { AwilixContainer } from "awilix"
+import { IParser } from "./abstract-parser"
+
+export type CsvParserContext = {
```

Comment:
question: ok so now I see why there is the LineContext earlier. So now, should that type be a composition of LineContext & { … } in order to keep the context consistent. In this context line correspond to the line number, but in the lineContext it is called lineNumber

Reply:
Well spotted! I think it's better to do as you suggest and reuse the `LineContext` type when definining the `CsvParserContext`

Reply:
👍